bigquery_etl/glean_usage/common.py

```diff
@@ -45,6 +45,35 @@ def write_sql(output_dir, full_table_id, basename, sql):
         f.write("\n")
 
 
+def write_dataset_metadata(output_dir, full_table_id):
+    """
+    Add dataset_metadata.yaml to public facing datasets.
+
+    Does not overwrite existing dataset_metadata.yaml files.
+    """
+    d = Path(os.path.join(output_dir, *list(full_table_id.split(".")[-2:])))
+    d.mkdir(parents=True, exist_ok=True)
```

Comment:
```suggestion
    d.parent.mkdir(parents=True, exist_ok=True)
```

Currently, we're writing out empty directories like `moz-fx-data-shared-prod/fenix/baseline_clients_daily`